### PR TITLE
replace django.core.urlresolvers with django.urls to appeace Django2.…

### PR DESCRIPTION
**Pull request by s-wirth**
…0 deprecations

### CODE REVIEW

File: defender/tests.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 from django.contrib.auth.models import User
 from django.contrib.auth.models import AnonymousUser
 from django.contrib.sessions.backends.db import SessionStore
-from django.core.urlresolvers import reverse
+from django.urls import reverse
```
Comment (kencochrane) [inline defender/tests.py]: This seems to be failing when run against old versions of django. Maybe we can do the following.


```
try:
    django.urls import reverse
except ImportError:
    django.core.urlresolvers import reverse
```

It tries to do it the new way, if it fails it falls back to the old way.

File: defender/views.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from django.shortcuts import render
 from django.http import HttpResponseRedirect
-from django.core.urlresolvers import reverse
+from django.urls import reverse
```
Comment (kencochrane) [inline defender/views.py]: Same comment as above.

**Comment by kencochrane** [comment]
I also noticed a similar change in #105 except your PR includes the tests.py as well.

**Comment by s-wirth** [comment]
@kencochrane Thank you very much for the comments! I also looked at the other PR you mentioned and added the updated install_requirements to my PR `setup.py`. Hope it works now :)

**Comment by btoueg** [comment]
I see that change requests are not the same for everyone ;)

Testing against Django 2.0 has not been added here !

**Comment by kencochrane** [comment]
@btoueg this PR was made after yours. Your PR already had those changes. I made the mistake and assumed your PR would be merged first and didn’t want to have duplicate code in more than one PR, to limit merge conflicts. Sorry for the trouble.

**Pull request merged**